### PR TITLE
i18n(es): Update `guides/customization.mdx` and `reference/configuration.md`

### PR DESCRIPTION
**Pull request by K1ri86**
#### What kind of changes does this PR include?
- Changes or translations of Starlight docs site content

#### Description
- What does this PR change? Give us a brief description.
- Update #752

### CODE REVIEW

**Comment by K1ri86** [comment]
> Thank you for the PR @K1ri86!
> 
> There were also two more changes to these files adding Telegram and RSS icons:
> 
> * [a0de12d](https://github.com/withastro/starlight/commit/a0de12d596ab2b1c0d79d6d63d2d6cb9fe6d2644)
> * [a8358df](https://github.com/withastro/starlight/commit/a8358df7849b342de342a4a2e88e019c39d5bbe8)
> 
> Would you be able to update this PR to include those as well?

Done 🚀

File: docs/src/content/docs/es/guides/customization.mdx [hunk 1]
```diff
@@ -188,7 +188,7 @@ defineConfig({
 
 Starlight cuenta con soporte incorporado para agregar enlaces a tus cuentas de redes sociales en el encabezado del sitio mediante la opción [`social`](/es/reference/configuration/#social) en la integración de Starlight.
 
-Actualmente, se admiten enlaces a Bitbucket, Codeberg, CodePen, Discord, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, Stack Overflow, Threads, Twitch, Twitter, y Youtube ¡Háznoslo saber en GitHub o Discord si necesitas soporte para otro servicio!
+Actualmente, se admiten enlaces a Bitbucket, Codeberg, CodePen, Discord, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, an RSS feed, Stack Overflow, Telegram, Threads, Twitch, Twitter, X, y Youtube ¡Háznoslo saber en GitHub o Discord si necesitas soporte para otro servicio!
```
[the remark this answers is not in the document]
Comment (K1ri86) [reply docs/src/content/docs/es/guides/customization.mdx:191]: > Just to double check, is "an RSS feed" actually properly translated or could have been forgotten?

I just made the change 👍🏻

**Review by kevinzunigacuellar — APPROVED** [review]
LGTM!

File: docs/src/content/docs/es/guides/customization.mdx [hunk 2]
```diff
@@ -188,7 +188,7 @@ defineConfig({
 
 Starlight cuenta con soporte incorporado para agregar enlaces a tus cuentas de redes sociales en el encabezado del sitio mediante la opción [`social`](/es/reference/configuration/#social) en la integración de Starlight.
 
-Actualmente, se admiten enlaces a Bitbucket, Codeberg, CodePen, Discord, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, Stack Overflow, Threads, Twitch, Twitter, y Youtube ¡Háznoslo saber en GitHub o Discord si necesitas soporte para otro servicio!
+Actualmente, se admiten enlaces a Bitbucket, Codeberg, CodePen, Discord, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, RSS feed, Stack Overflow, Telegram, Threads, Twitch, Twitter, X, y Youtube ¡Háznoslo saber en GitHub o Discord si necesitas soporte para otro servicio!
```
Comment (dreyfus92) [inline docs/src/content/docs/es/guides/customization.mdx:191]:
```suggestion
Actualmente, se admiten enlaces a Bitbucket, Codeberg, CodePen, Discord, GitHub, GitLab, Gitter, Instagram, LinkedIn, Mastodon, Microsoft Teams, un flujo RSS, Stack Overflow, Telegram, Threads, Twitch, Twitter, X, y Youtube ¡Háznoslo saber en GitHub o Discord si necesitas soporte para otro servicio!
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Awesome job @K1ri86, I left you a minor suggestion 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by delucis — APPROVED** [review]
Thanks again @K1ri86 and everyone who reviewed. Great team work 🙌

**Pull request merged**